src/main/java/frc/robot/subsystems/TurretSubsystem.java

```diff
@@ -132,6 +150,7 @@ public TurretSubsystem(RobotStateManager robotStateManager) {
     turretPIDController.setIZone(Constants.TurretConstants.KIZ);
 
     zeroTurret();
+    SmartDashboard.putData("zero turret", zeroZeroing());
```

Comment:
Can we add this to the shuffleboard tab for the turret instead?

Reply:
based off of my reading, this is the official way to get it to be  shuffle board

ref: https://docs.wpilib.org/en/stable/docs/software/dashboards/shuffleboard/advanced-usage/shuffleboard-commands-subsystems.html

Reply:
But it puts it in the SmartDashboard tab which is cluttered. 
```suggestion
      turretTab.add("zero turret", zeroZeroing());
```
This works to put it on the turret tab

Reply:
okay, couldn't find that in the documentation